smarty/templates/main.tpl

```diff
@@ -528,7 +528,7 @@
                                         </div>
                                     {/if}
                                 {/if}
-                                <div id="kgkjgkjg">
+                                <div id="workspace">
```

Comment:
There's a (very slight) possibility that a module might use the id "workspace" for something. Can we call it something more specific like "lorisworkspace"?